plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java

```diff
@@ -2292,6 +2287,23 @@ protected void doSetInput(IEditorInput editorInput) throws CoreException {
         editorImage = new Image(Display.getCurrent(), baseEditorImage, SWT.IMAGE_COPY);
     }
 
+    private void accomplishEditorInputInitialization(@NotNull IEditorInput editorInput) {
+        DBPDataSourceContainer oldDataSource = SQLEditor.this.getDataSourceContainer();
```

Comment:
Why do you use explicit `this` everywhere?

Reply:
It doesn't work without `this` here.
Also I like the general approach of clearly specifying that the object comes from the current class scope